dev/tests/js/jasmine/tests/lib/mage/dropdown.test.js

```diff
@@ -270,29 +270,34 @@ define([
             jasmine.clock().uninstall();
         });
 
-        it('check if the title bar is prevented from being created', function () {
-            var dialog = $('<div/>'),
-                uiClass = '.ui-dialog',
-                ui;
+        // Backend jquery-ui version has a diferent behavior on create that will add ui-dialog-titlebar anyway.
```

Comment:
@omiroshnichenko looks like I've found a "stopper". 

One of the tests from dropdown.test.js was failing on backend test due to the differences with jquery-ui version. Frontend version won't add on create ui-dialog-titlebar while backend will do so.

I decided to apply a selective expectation based on jquery-ui version instead of re-designing the jasmine configuration for lib folder and applying structural changes to do so. (ie: lib/mage/frontend/**/*.test.js, lib/mage/backend/**/*.test.js ... )

What do you think?

Reply:
Your solution looks good for me. Re-designing jasmine configuration for lib folder it is incorrect because lib folder can be used in both areas.